### DISCUSSION TITLE
So how well is this working out?

### DISCUSSION THREAD

**Discussion by Zipties** [ESPresense/ad-espresense-ips#24] [General]
Just curious how well this concept is working out in practice for people. Are you able to actually get  room level accuracy with triangulation, or is it mostly whatever weird BLE signal reflection is taking over?

is this usable at this point or more still a poc?

**Comment by DTTerastar** [comment]
The latest changes have vastly improved things.  But it's still fairly POC since it really only supports one device being plotted.
